packages/http-helper/src/http_method.ts

```diff
@@ -58,3 +58,14 @@ export const TRACE = 'TRACE';
  *  - https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PATCH
  */
 export const PATCH = 'PATCH';
+
+export type HttpMethods =
```

Comment:
I hesitate to use `HttpMethods` name it's easily to typo with `HttpMethod.` 

Instead, how about `HttpMethodType`? (it's also redundant. But explict).